web/src/Setting.js

```diff
@@ -1131,11 +1131,35 @@ export function renderLogo(application) {
   }
 }
 
+export function isPasswordEnabled(application) {
+  if (application) {
+    return application.signinMethods.filter(item => item.name === "Password").length > 0;
+  } else {
+    return false;
+  }
```

Comment:
`return application?.signinMethods?.filter(item => item.name === "Password").length > 0;`
I found the signinMethods may be null when I upgraded from an older version.